app/Models/Themes.php

```diff
@@ -95,6 +96,7 @@ public static function alt($name) {
 			'rss' => '☄',
 			'unread' => '☐',
 			'refresh' => '🔃',	//↻
+			'screwdriver' => '🔩',
```

Comment:
I wonder if this is appropriated. I understand that you want to display it along with actions that update the cache. My concern is about how users will understand that icon.

Reply:
Yes, I agree. I didn't find any screwdriver emoticons, and as it was just alt (I added a real screwdriver svg icon)…

Do you have better idea ? Perhaps use the `⚙︎` or `⚙️` as alternative ? Or remove `screwdriver` in a whole, and use `configure` instead ?

Reply:
The thing is I do not have a better idea. :/
I am just wondering if we won't end in the unknown icon hell.

Reply:
Yes, I'm aware of this unknown icon hell. And I'm not a UX expert… Perhaps `🛠` or `🔧`, and rename the icon as "maintenance", or `repair`, or `housekeeping` (and keep the screwdriver as the svg icon ?).

Look a bit more coherent, generic but still somehow self-explanatory ? For sure, the icon can't be used alone, but…

Reply:
What exactly is the goal you wish to achieve by adding this screwdriver icon to these two buttons?

Reply:
Decoration ? Relief ? Like all button icon…

Reply:
All button icons where? The only one I see is the standard refresh icon on the endearingly named actualize button. ;-) (Actualize is French for refresh; it's English for making something actual/real.) HIG guidelines typically say something like [this](https://developer.apple.com/design/human-interface-guidelines/macos/menus/menu-anatomy/#using-symbols-in-menus):

> Avoid using custom symbols in menus. People are familiar with the standard symbols. Using nonstandard symbols introduces visual clutter and could confuse the user.

Reply:
> we know when we should add an icon, and when we should not.

When in doubt, don't.

Reply:
My doubt was not that strong :p 

I agree from Apple's HIG guidelines, but most of the time, I try to follow what is currently done, at least for homogeneity.

By example, in this panel:
- There are icons for side actions (like the pointer icon for Website URL & Feed URL, and Actualize)
- There are icons for help text (I thought it was a button or a mouse-over tooltip, the first time).
- There is not icons for main actions, like submit or cancel.
- And there is some intruder, like "Check the validity of the feed" or "Delete all articles".

My decision was just trying to homogenize by "category".

Reply:
> There are icons for side actions (like the pointer icon for Website URL & Feed URL, and Actualize)

These are a different category.

> There are icons for help text (I thought it was a button or a mouse-over tooltip, the first time).

I believe it's a lifebuoy. Definitely not very clear.

> And there is some intruder, like "Check the validity of the feed" or "Delete all articles".

Not an intruder, perfectly consistent.

Those new buttons need to be blue to be consistent with the current UI. (For better or worse.)

Reply:
> Not an intruder, perfectly consistent.

You created them ? :p

> Those new buttons need to be blue to be consistent with the current UI. (For better or worse.)
 
But then, why "Check the validity of the feed" is not blue ? 

Well, we don't have the same categorizing algorithm, I think. But that's okay, I will turn my buttons blue, and remove the icons. But I will stay the preview button as-it, I think an eye is better than a textual "Preview".

Reply:
> You created them ? :p

Nope, you might say I created surprisingly little. But I didn't get involved until post 1.0. ;-)

(You have me to thank for all the automated CI comments on your PHP, JS and CSS, for example. :-D)

> But then, why "Check the validity of the feed" is not blue ?

The way I see it, blue signifies an action that will have lasting consequences. (It will submit the form.)

Reply:
> (You have me to thank for all the automated CI comments on your PHP, JS and CSS, for example. :-D)

Thank you ! (no, seriously, we have a lot of disagreement, but I really appreciate all your comments, and the effort you put in the review !).

Reply:
Fixed.